packages/app-store/googlecalendar/lib/CalendarService.test.ts

```diff
@@ -1,12 +1,11 @@
-import prismaMock from "../../../../tests/libs/__mocks__/prismaMock";
```

Comment:
Using prismock instead that allows testing with an in memory DB instead of mocking the calls.